game/src/machines/machine.rs

```diff
@@ -53,11 +52,16 @@ impl Machine {
         )
     }
 
-    pub fn new(gs: &GameState, name: String, hit_box: Rect, interaction_area: Rect) -> Self {
+    pub fn new(
+        gs: &GameState,
+        name: String,
+        hit_box: Rect,
+        interaction_area: Rect,
+    ) -> RLResult<Self> {
         info!("Creating new machine: name: {}", name);
```

Comment:
Dann kann man hier doch auch den Value in die Klammer schreiben oder?

Reply:
Ne nur bei fmt!(Vielleicht bald auch woanders, das weiß ich nicht)